### PR TITLE
Reduce Sqlite nuget dependencies

### PR DESCRIPTION
**Pull request by dannyvv**

### CODE REVIEW

File: config.dsc [hunk 1]
```diff
@@ -121,10 +121,7 @@ config({
                 { id: "Newtonsoft.Json", version: "11.0.2" },
                 { id: "Newtonsoft.Json", version: "10.0.3", alias: "Newtonsoft.Json.v10" },
                 { id: "Newtonsoft.Json.Bson", version: "1.0.1" },
-                { id: "System.Data.SQLite", version: "1.0.109.2" },
```
Comment (almiliMSFT) [inline config.dsc]: how come that all of a sudden we don't need this?

Comment (Kr1sso) [reply config.dsc]: I guess it’s because you disabled DB init for aria 😀

File: Public/Src/App/Bxl/Bxl.dsc [hunk 2]
```diff
@@ -75,7 +75,6 @@ namespace Main {
             importFrom("BuildXL.FrontEnd").CMake.dll,
             importFrom("BuildXL.FrontEnd").Sdk.dll,
             importFrom("Newtonsoft.Json").pkg,
-            importFrom("System.Data.SQLite.Core").pkg,
```
Comment (almiliMSFT) [inline Public/Src/App/Bxl/Bxl.dsc]: note that this package has the `.Core` suffix which is the one you didn't remove from `config.dsc` (I'm not sure if this comment carries any significance though :))

File: config.dsc (hunk 1 above)
Comment (almiliMSFT) [reply config.dsc]: only on Windows; on macOS it should be just the same as before

Comment (dannyvv) [reply config.dsc]: It is not used at all, there were only references to the .Core pacakge

---
In reply to: [310282058](https://github.com/microsoft/BuildXL/pull/696#discussion_r310282058) [](ancestors = 310282058)

Comment (dannyvv) [reply config.dsc]: Note used, see other comment

---
In reply to: [310281164](https://github.com/microsoft/BuildXL/pull/696#discussion_r310281164) [](ancestors = 310281164)

File: Public/Src/App/Bxl/Bxl.dsc (hunk 2 above)
Comment (dannyvv) [reply Public/Src/App/Bxl/Bxl.dsc]: our main exe doesn't need the dll to compile, so I removed it.

---
In reply to: [310281736](https://github.com/microsoft/BuildXL/pull/696#discussion_r310281736) [](ancestors = 310281736)

**Pull request merged**